tdrs-backend/tdpservice/users/api/middleware.py

```diff
@@ -0,0 +1,34 @@
+"""Handle authorization calls globally."""
+import datetime
+from django.conf import settings
+
+
+class AuthUpdateMiddleware:
```

Comment:
This will update the HTTP Only cookie with a new expiration date/time on every request